projects/shared-modules/hms-video/components/hms-video/local-preview/local-preview.component.ts

```diff
@@ -39,10 +39,13 @@ export class LocalPreviewComponent implements OnInit, OnDestroy {
   }
 
   getMediaPermissions(): void {
+    const microphone = 'microphone' as PermissionName;
+    const camera = 'camera' as PermissionName;
+
     this.subscriptions.push(
       combineLatest(
-        this.localMediaService.getMediaPermissions('camera'),
-        this.localMediaService.getMediaPermissions('microphone'),
+        this.localMediaService.getMediaPermissions(camera),
+        this.localMediaService.getMediaPermissions(microphone),
```

Comment:
Please check this one too @ArshdeepGrover

Reply:
Works perfectly!